learnAndCodeFinalProject/src/main/java/org/learnAndCode/RecommendationEngine.java

```diff
@@ -0,0 +1,88 @@
+package org.learnAndCode;
+
+import org.learnAndCode.Database.DBConnection;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecommendationEngine {
+
+    public static List<MenuItem> getRecommendations() {
+        List<MenuItem> recommendations = new ArrayList<>();
+
+        String query = "SELECT item_id, item_name, rating, review " +
```

Comment:
The name of the variable coul have been more specific on the basis of what this query is written for.